### DISCUSSION TITLE
Partition Coefficients (PCS) Definition?

### DISCUSSION THREAD

**Discussion by jmgallo** [Open-Systems-Pharmacology/Forum#834] [Questions & Problems]
Hi,
It's unclear how the PCs - reported as Interstitial:Plasma and Intracellular:Plasma - are defined and used in the ODEs. For example, for a 3-subcompartment (vascular, interstitial fluid and intracellular) tissue you want 2 PCs; one defined as the unbound fraction in plasma/unbound fraction in the interstitial fluid, and the second PC, defined as the PC as unbound fraction in the interstitial fluid/intracellular compartment.  These PCs are used in transport or intercompartment flux terms in the ODEs.  For example, the PC defined as unbound fraction in the interstitial fluid/intracellular compartments could be obtained by dividing the 2 PC provided by PK-Sim. Is that how you envision their use? Further, in a blood-flow limited tissue, the PC (assuming linear binding) is = unbound fraction in blood (or plasma)/unbound fraction in the tissue. Can this be extracted from PK-Sim.  
Thanks, jim

**Comment by StephanSchaller** [comment]
Hi Jim,

you can find a more detailed explanation of PCs in the manual:
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-definition-and-work-flow#adme-properties

Also, if you export your simulation to MoBi, it will likely get clearer how the ODEs are set up, or at least how the rates (e.g. transports) that make up the ODEs are set up.

Best,
Stephan

**Comment by jmgallo** [comment]
Hi Stephan,
Thanks, but it's still unclear how the PK-Sim generated PCs (yes, I see the 5 methods of calculation) are used in the ODEs. I can't load my PK-Sim model into MoBi to investigate as you suggest; clearly I don't use these programs. If you would have time for a Zoom later this week I can share my model.

Thanks and regards, jim

**Comment by StephanSchaller** [comment]
Hi Jim,

excuse the delay in responding.
Sure, let me know a good time: stephan.schaller@esqlabs.com

Best,
Stephan
